fedmsg_meta_fedora_infrastructure/fasshim.py

```diff
@@ -133,13 +131,20 @@ def _add_to_cache(users):
     global _fas_cache
 
     for user in users:
-        nicks = user.get('ircnicks', [])
-        for nick in nicks:
+        nick = user.get('ircnick')
```

Comment:
I don't think a user has a `ircnick` property, it's always the multi-valued `ircnicks`.

Reply:
This is the original check that I just moved, its the FAS attribute `ircnick` not FASJSON's.